src/fhirtypes/StructureDefinition.ts

```diff
@@ -575,7 +575,7 @@ export class StructureDefinition {
       // fixValue will throw if it fails, but skip the check if value is null
       if (value != null) {
         // since we don't actually keep the clone, argument for exactly should not matter
```

Comment:
Question - is this comment not really true anymore? Should we update it to say why we need `exactly` to be `true`?

Reply:
Oops! Good catch. I remember seeing that and thinking I should fix it, but here we are.

Reply:
I've been there countless times, so I support you.